src/components/DeviceStatusGauge.tsx

```diff
@@ -0,0 +1,73 @@
+import filter = require('lodash/filter');
+import * as React from 'react';
+import { compose } from 'recompose';
+// @ts-ignore
```

Comment:
What is this for?

Reply:
@sradevski Would it be simple enough to add a small typings file here https://github.com/balena-io-modules/rendition/tree/master/src/typings ?
I'd prefer to do that, even if we are going to remove this component anyway